gradio/components/image.py

```diff
@@ -165,7 +188,7 @@ def __init__(
         )
 
     def preprocess(
-        self, payload: FileData | None
+        self, payload: ImageData | None
```

Comment:
I believe this should be:
```suggestion
        self, payload: ImageData | Base64ImageData | None
```